src/reducers.js

```diff
@@ -63,23 +64,37 @@ const perPage = (state = PER_PAGE) => {
   return state
 }
 
-// const errorReport = (state = '', action) => {
-//   switch (action.type) {
-//     case REPORT_ERROR:
-//       console.log(`${action.errorCode}: ${action.errorMessage}`)
-//       return `${action.errorCode}: ${action.errorMessage}`
-//     default:
-//       return state
-//   }
-
-// }
-
-const rootReducer = combineReducers({
+const appReducer = combineReducers({
```

Comment:
So now I want to get in the middle of your reducer code and intercept asynchronous actions separately. I'm therefore going to take your `rootReducer` code and call it `appReducer`. We'll call that if there aren't any async actions.